flink-tests/src/test/java/org/apache/flink/runtime/jobmaster/JobMasterITCase.java

```diff
@@ -47,22 +47,23 @@
 
     @Test
     public void testRejectionOfEmptyJobGraphs() throws Exception {
-        MiniCluster miniCluster =
-                new MiniCluster(
-                        new MiniClusterConfiguration.Builder()
-                                .setNumTaskManagers(1)
-                                .setNumSlotsPerTaskManager(1)
+        MiniClusterResource miniCluster =
```

Comment:
What's the benefit here?

Reply:
causes a random port to be used for the rest api